db/changelog/1.0.0-create-schema-and-moo-table.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
When we want to make a new database change to we create a new one of these files manually?

Are there any rules around  how you increase the change version? The format seems to suggest we do semantic versioning? Do you know if this is used by anything (e.g. does some part of liquibase treat major version increases differently somehow)?

Reply:
Yeah, we will need to create one of these each time we change the database and reference it in the main changelog file. Would it be worth documenting the process in the readme/confluence?

For the change version, do you mean the version in the id? I think we decided to add the change version in for our own clarity, liquibase doesn't do anything with it

Reply:
We agreed to move to more simple `001` version numbers but we have a problem in that these changes are already applied to our dev database and we don’t have access to clear it out 🤦 we’re not going to consider this a blocker for merging but we’re going to think about how to fix this tomorrow.